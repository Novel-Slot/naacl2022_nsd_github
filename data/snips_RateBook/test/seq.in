rate the current novel four of 6 stars
i rate shadow of suribachi at five stars
rate lamy of santa fe 5 of 6 stars
rate cuisines of the axis of evil and other irritating states one out of 6
rate the chronicle ten from tomorrow a 2
rate the book an appeal from the new to the old whigs a 0
rate this current novel 1 stars
i rate secret water as a 4
give 1 out of 6 points to this novel
rate this textbook a zero
rate awaiting strange gods: weird and lovecraftian fictions a 1
give 1 point to current textbook
rate the current chronicle a zero
a day no pigs would die deserves a best rating of 6 and a value of 4
i would rate that old ace in the hole one stars and a best rating of 6
add the rating for this current series a four out of 6 points
rate the beggar of volubilis 1 out of 6
give the current book five stars out of 6
rate current essay a zero
the last hawk gets a total of 3 out of 6 stars from me
rate the current chronicle five stars
rate this novel five of 6
my rating for the eiffel tower and other mythologies is 0 out of 6 stars
rate this textbook 2 out of 6
give this current book zero out of 6
rate this album 5 points
i want to give the current textbook 0 out of 6 stars
i would rate the persistence of vision 1 stars and a best rating of 6
give the current album a five
the current essay gets four points
rate the book english grammar in use a five
i want to give a mortal flower a two
rate the little book four stars
rate the current textbook one of 6 stars
give 2 out of 6 points to the following textbook
award this current novel 0 points
rate this book four of 6 points
cock-a-doodle-doo was awful  i m giving it a 0 out of 6
i would give french poets and novelists a best rating of 6 and a value of three
i want to give the chronicle zombie bums from uranus 3 points
i would give this current novel 2 stars with a best rating of 6
rate this album a 2
i rate doom 3: worlds on fire a 1 of 6
rate in the eyes of mr  fury zero of 6
i give the phishing manual four stars out of 6
the chronicle charlie peace earns 4 stars from me
rate if tomorrow comes 2 of 6 stars
the book history by contract is rated five stars in my opinion
rate this novel a 3
give private games 3 stars out of 6
i rate the chronicle  son of the tree with four of 6 points
rate maps for lost lovers 1 of 6
rate the manxman 5 out of 6
give 4 points to this novel
rate this essay a two out of 6
give this chronicle a 4
rate the key word and other mysteries 4 of 6
i want to rate the ingenuity gap 3 out of 6
rate the current chronicle series 3 out of 6 points
put four rating on the raging quiet
rate the lie tree five
give 2 stars to the doom brigade
rate this series 2 out of 6
rate the current novel four out of 6 points
for the current saga i rate 2 of 6 stars
rate the current book a three
rate this novel 0 of 6 stars
rate canto for a gypsy two of 6 stars
this textbook gets a two
rate this textbook a 1
rate a taste of blackberries  a three
rate the saint in trouble 1 of 6
rate this book three points
i am giving this current book album 0 out of 6 stars
rate wielding a red sword 0 stars
the current textbook gets a 2 rating
rate this book titled the improvisatore five stars
i give this book dictionary of the english language a 4 rating
rate this album 0 points out of 6
rate this album two out of 6
