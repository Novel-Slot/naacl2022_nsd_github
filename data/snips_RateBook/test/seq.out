O O B-object_select B-object_type B-rating_value O B-best_rating B-rating_unit
O O B-object_name I-object_name I-object_name O B-rating_value B-rating_unit
O B-object_name I-object_name I-object_name I-object_name B-rating_value O B-best_rating B-rating_unit
O B-object_name I-object_name I-object_name I-object_name I-object_name I-object_name I-object_name I-object_name I-object_name I-object_name B-rating_value O O B-best_rating
O O B-object_part_of_series_type B-object_name I-object_name I-object_name O B-rating_value
O O B-object_type B-object_name I-object_name I-object_name I-object_name I-object_name I-object_name I-object_name I-object_name I-object_name O B-rating_value
O O B-object_select B-object_type B-rating_value B-rating_unit
O O B-object_name I-object_name O O B-rating_value
O B-rating_value O O B-best_rating B-rating_unit O B-object_select B-object_type
O B-object_select B-object_type O B-rating_value
O B-object_name I-object_name I-object_name I-object_name I-object_name I-object_name I-object_name O B-rating_value
O B-rating_value O O B-object_select B-object_type
O O B-object_select B-object_part_of_series_type O B-rating_value
B-object_name I-object_name I-object_name I-object_name I-object_name I-object_name O O O O O B-best_rating O O O O B-rating_value
O O O B-object_name I-object_name I-object_name I-object_name I-object_name I-object_name B-rating_value B-rating_unit O O O O O B-best_rating
O O O O O B-object_select B-object_part_of_series_type O B-rating_value O O B-best_rating B-rating_unit
O B-object_name I-object_name I-object_name I-object_name B-rating_value O O B-best_rating
O O B-object_select B-object_type B-rating_value B-rating_unit O O B-best_rating
O B-object_select B-object_type O B-rating_value
B-object_name I-object_name I-object_name O O O O B-rating_value O O B-best_rating B-rating_unit O O
O O B-object_select B-object_part_of_series_type B-rating_value B-rating_unit
O B-object_select B-object_type B-rating_value O B-best_rating
O O O B-object_name I-object_name I-object_name I-object_name I-object_name I-object_name O B-rating_value O O B-best_rating B-rating_unit
O B-object_select B-object_type B-rating_value O O B-best_rating
O O B-object_select B-object_type B-rating_value O O B-best_rating
O B-object_select B-object_type B-rating_value B-rating_unit
O O O O O B-object_select B-object_type B-rating_value O O B-best_rating B-rating_unit
O O O B-object_name I-object_name I-object_name I-object_name B-rating_value B-rating_unit O O O O O B-best_rating
O O B-object_select B-object_type O B-rating_value
O B-object_select B-object_type O B-rating_value B-rating_unit
O O B-object_type B-object_name I-object_name I-object_name I-object_name O B-rating_value
O O O O B-object_name I-object_name I-object_name O B-rating_value
O B-object_name I-object_name I-object_name B-rating_value B-rating_unit
O O B-object_select B-object_type B-rating_value O B-best_rating B-rating_unit
O B-rating_value O O B-best_rating B-rating_unit O O B-object_select B-object_type
O O B-object_select B-object_type B-rating_value B-rating_unit
O B-object_select B-object_type B-rating_value O B-best_rating B-rating_unit
B-object_name O O O O O O O B-rating_value O O B-best_rating
O O O B-object_name I-object_name I-object_name I-object_name O O O O B-best_rating O O O O B-rating_value
O O O O O B-object_part_of_series_type B-object_name I-object_name I-object_name I-object_name B-rating_value B-rating_unit
O O O O B-object_select B-object_type B-rating_value B-rating_unit O O O O O B-best_rating
O B-object_select B-object_type O B-rating_value
O O B-object_name I-object_name I-object_name I-object_name I-object_name O B-rating_value O B-best_rating
O B-object_name I-object_name I-object_name I-object_name I-object_name I-object_name B-rating_value O B-best_rating
O O B-object_name I-object_name I-object_name B-rating_value B-rating_unit O O B-best_rating
O B-object_part_of_series_type B-object_name I-object_name O B-rating_value B-rating_unit O O
O B-object_name I-object_name I-object_name B-rating_value O B-best_rating B-rating_unit
O B-object_type B-object_name I-object_name I-object_name O O B-rating_value B-rating_unit O O O
O B-object_select B-object_type O B-rating_value
O B-object_name I-object_name B-rating_value B-rating_unit O O B-best_rating
O O O B-object_part_of_series_type B-object_name I-object_name I-object_name I-object_name O B-rating_value O B-best_rating B-rating_unit
O B-object_name I-object_name I-object_name I-object_name B-rating_value O B-best_rating
O B-object_name I-object_name B-rating_value O O B-best_rating
O B-rating_value B-rating_unit O B-object_select B-object_type
O B-object_select B-object_type O B-rating_value O O B-best_rating
O B-object_select B-object_part_of_series_type O B-rating_value
O B-object_name I-object_name I-object_name I-object_name I-object_name I-object_name B-rating_value O B-best_rating
O O O O B-object_name I-object_name I-object_name B-rating_value O O B-best_rating
O O B-object_select B-object_part_of_series_type I-object_part_of_series_type B-rating_value O O B-best_rating B-rating_unit
O B-rating_value O O B-object_name I-object_name I-object_name
O B-object_name I-object_name I-object_name B-rating_value
O B-rating_value B-rating_unit O B-object_name I-object_name I-object_name
O B-object_select B-object_part_of_series_type B-rating_value O O B-best_rating
O O B-object_select B-object_type B-rating_value O O B-best_rating B-rating_unit
O O B-object_select B-object_part_of_series_type O O B-rating_value O B-best_rating B-rating_unit
O O B-object_select B-object_type O B-rating_value
O B-object_select B-object_type B-rating_value O B-best_rating B-rating_unit
O B-object_name I-object_name I-object_name I-object_name B-rating_value O B-best_rating B-rating_unit
B-object_select B-object_type O O B-rating_value
O B-object_select B-object_type O B-rating_value
O B-object_name I-object_name I-object_name I-object_name O B-rating_value
O B-object_name I-object_name I-object_name I-object_name B-rating_value O B-best_rating
O B-object_select B-object_type B-rating_value B-rating_unit
O O O O B-object_select B-object_type I-object_type B-rating_value O O B-best_rating B-rating_unit
O B-object_name I-object_name I-object_name I-object_name B-rating_value B-rating_unit
O B-object_select B-object_type O O B-rating_value O
O O B-object_type O B-object_name I-object_name B-rating_value B-rating_unit
O O O B-object_type B-object_name I-object_name I-object_name I-object_name I-object_name O B-rating_value O
O B-object_select B-object_type B-rating_value B-rating_unit O O B-best_rating
O B-object_select B-object_type B-rating_value O O B-best_rating
